packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -196,7 +196,10 @@ class BottomSheetCell extends Component {
 				<TouchableOpacity
 					onPress={ onClear }
 					style={ styles.clearIconContainer }
-					testID="clear-button"
+					accessibilityLabel={ __( 'Clear Button' ) }
+					accessibilityHint={ __(
+						'Will clear the value of the text input when pressed'
```

Comment:
I checked other hint messages ([reference](https://github.com/WordPress/gutenberg/blob/a8061c1db1e4252fb14d0ee033cf3340d694b114/packages/block-library/src/missing/edit.native.js#L91)) and I noticed that we also include the action to be made by the user, so I'm wondering if we should update the message with something like:
```Tap here to clear the value of the text input```

What do you think?

Reply:
I think it's a great idea -- I've updated the accessibility hint to use the imperative mood.